src/components/SearchBox.js

```diff
@@ -41,28 +41,42 @@ class SearchBox extends Component {
             style={{
               // make sure input field has same height as search button
               padding: "6px",
+              paddingLeft: "35px"
               // right corners will touch button, remove these rounded corners added by FormStyles
-              borderTopRightRadius: "0px",
-              borderBottomRightRadius: "0px"
+              // borderTopRightRadius: "0px",
```

Comment:
deze 2 uitgecommente regels kunnen weg, evenals de comment erboven ;)

Reply:
Note: als de zoekterm even groot is als de textbox zonder vergrootglas, valt de tekst onder het kruisje (om de tekstbox te clearen)

Reply:
paddingRight: "25px"